src/main/scala/cromwell/engine/io/gcs/GoogleConfiguration.scala

```diff
@@ -12,6 +12,7 @@ import scalaz._
 sealed trait GoogleCromwellAuthMode
 final case class ServiceAccountMode(accountId: String, pemPath: String) extends GoogleCromwellAuthMode
 final case class UserMode(user: String, secretsFile: String, datastoreDir: String) extends GoogleCromwellAuthMode
+final case object ApplicationDefaultMode extends GoogleCromwellAuthMode
```

Comment:
@tovanadler I was just lamenting that various authoritative sources of pedantry are in conflict. It's effectively useless as @mcovarr states